src/analysis/stack_or_heap_enclosing.ml

```diff
@@ -0,0 +1,67 @@
+open Std
+
+let log_section = "stack-or-heap-enclosing"
+let { Logger.log } = Logger.for_section log_section
```

Comment:
Is there anything interesting to log? (I don't have anything particular in mind, just wanted to note that this is unused.)

Reply:
We discussed in-person and decided to leave declarations exposed in case a different part of merlin decides to log something about our `stack-or-heap`.